query/test/unit/test_links_file_transform.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+from mock import patch
+
+from query.lib.config import Config
+from query.lib.db.database import Tables, PostgresDatabase
+from query.lib.etl.links_file_transform import LinksFileTransform
+from test import mock_links, truncate_tables
+
+
+class TestLinksTransform(unittest.TestCase):
+    db = PostgresDatabase(Config.test_database_uri)
```

Comment:
If you return a transformed data structure from the `LinksTransformer` instead of passing the tables in, you could remove all database concerns from this test and just interact with the data structure itself. Really what you care about here is testing the functionality of the `LinksTransformer` which you can do without concern of the storage layer.

Reply:
👍